### DISCUSSION TITLE
error 'No module named 'ase.filters'' when i try to relax the structure

### DISCUSSION THREAD

**Discussion by kaijiz** [CederGroupHub/chgnet#134] [Q&A] [answered]
Hi, when i tried to use the example code to try relaxing the structures. I met this error. I don't know how to fix it, could anyone help me.
![Screen Shot 2024-03-04 at 4 19 50 PM](https://github.com/CederGroupHub/chgnet/assets/126427660/09d92569-37b7-4256-bac5-bf24bf7d6753)
![Screen Shot 2024-03-04 at 4 20 53 PM](https://github.com/CederGroupHub/chgnet/assets/126427660/7c34751c-232c-44c6-83ad-d928020bb3f8)

**Comment by bowen-bd** [comment, marked as answer]
see https://github.com/CederGroupHub/chgnet/issues/127
and https://github.com/CederGroupHub/chgnet/issues/118
